test/analyzer.js

```diff
@@ -26,7 +26,7 @@ describe('Analyzer', function () {
 
   it('should support stats files with all the information in `children` array', async function () {
     generateReportFrom('with-children-array.json');
-    await expectValidReport();
+    return expectValidReport();
```

Comment:
Was this change required, or was increasing the timeout value enough? I see that there are some other specs which have the `await expectValidReport()` and they don't seem to be failing now.

Reply:
timeout here didn't change, just know that in some cases mocha doesn't handle await properly and the easy way is to just return the async function

Reply:
seems like it didn't solve the issue, increasing timeout

Reply:
Should we then change the other `await expectValidReport` usages in this file, too? The test files should go through babel, if I'm not mistaken, and get transpiled so that `async`/`await` would work in the mocha tests...